### ISSUE TITLE
Setup a page template

### ISSUE THREAD

**Issue by sw4d** [sw4d/no_trump_news#3]
We need a template that can be used for viewing links to NTN articles. 

- Should be dark contrast. 
- Should be accessible and qualify WCAG 2.2 AAA
-  Should contain a header, some kind of navigation outlining
- - about
- - privacy
